server.ts

```diff
@@ -1,18 +1,17 @@
-const
-    { connect } = require('./config/db.server.config'),
-    express = require('./config/express.server.config');
+import { connect } from './config/db.server.config'
+import express from './config/express.server.config';
 
 // Express.js application object
 const app = express();
 
 // Server port - use environment variable PORT or enter here
-const PORT = process.env.PORT;
+const PORT = process.env.PORT ?? 4200;
 
 // Database name - use environment variable DATABASE_NAME or enter here
-const databaseName = process.env.DATABASE_NAME;
+const databaseName = process.env.DATABASE_NAME as string;
```

Comment:
doesn't seem to be used & don't envvars always cast as string (is the explicit cast required)?